src/relay.js

```diff
@@ -144,8 +144,20 @@ export function sequelizeConnection({
     }
   };
 
-  let orderByAttribute = function (orderBy) {
-    return orderBy[0][0];
+  let orderByAttribute = function (orderAttr, args) {
+    return typeof orderAttr === 'function' ? orderAttr(args) : orderAttr;
+  };
+
+  let orderByDirection = function (orderDirection, args) {
+    if (args.last) {
+      orderDirection = orderDirection.indexOf('ASC') >= 0
+              ? orderDirection.replace('ASC', 'DESC')
+              : orderDirection.replace('DESC', 'ASC');
+      orderDirection = orderDirection.indexOf('LAST') >= 0
+              ? orderDirection.replace('LAST', 'FIRST')
+              : orderDirection.replace('FIRST', 'LAST');
```

Comment:
You're inverting `NULLS FIRST` to `NULL LAST` and vice versa? Not sure that's the best way to go, hmm.